Проверяем макет по критериям

TV37 opened the pull request:
+ заменяет в главном слайдере таблицу на флекс -бокс

---
:mortar_board: [Проверяем макет по критериям](https://up.htmlacademy.ru/htmlcss-individual/1/user/2292795/tasks/12)

:boom: https://htmlacademy-htmlcss.github.io/2292795-device-1/8/

TV37 committed:
Проверяет главную стр в Perf pix

+ заменяет в главном слайдере таблицу на флекс -бокс

TV37 committed:
Проверяет стр Каталога в Perfect Pixel

Director099 commented on index.html:
```diff
@@ -36,8 +36,11 @@
               <input class="navigation-form-input" id="search-form" type="search" name="serch"
                 placeholder="Поиск по сайту">
+              <!-- после наведеня на форму поиска -->
+              <!--<input class="navigation-form-input" id="search-form" type="search"  name="serch" placeholder="Текст поиска">-->
```
Можно удалить т.к. текст меняться не будет, в макете типо текст ввели

Director099 reviewed (commented):
http://joxi.ru/J2bKZQ5cMPKKKA - добавь для боди `min-width: 1190px` чтоб при меньших экранах не ломались блоки

Director099 commented:
1) Пример стайлгайд кнопки https://jsbin.com/jekipatoyi/edit?html,css,output
2) Пример стайлгайда формы https://jsbin.com/nuvalapuyo/edit?html,css,output
3) Удали пожалуйста цвет у фокуса, он лишний в макете нету http://joxi.ru/a2XJ0oetpqPa7r
4) http://joxi.ru/LmGdRo3cZ7WYG2 реализация не верная

`.delivery-rare :hover { background-color: #FFC700; }` - это свойство лишнее т.к. ты все охватываешь лучше убрать

Решение:

```
.delivery-rare:hover .delivery-rare-image { 
  background-color: #FFC700; 
}

.delivery-rare:hover .devilery-link::before { 
  background-color: #FFC700; 
}
```

5) Ошибки валидатора, проверь пожалуйста
6) когда делаешь фокуса у тебя часто проиходит сдвиг из за border, используй outline
7) Добавь фавикон https://up.htmlacademy.ru/htmlcss-individual/1/criteries#HTML-08

TV37 committed:
Дз критерии продолжение

1) удаляет лишние файлы
2) переделывает модальное окно
3) проверяет в pixel perfect поповер, модальное окно, тултип.
4) исправляет body width
5) начинает работать над ошибками и критериями

TV37 committed:
Элементы активности

1) Флав- иконки
2) элементы активности heder переделаны - убраны картинки в blackground-img
3) Доставка
4) переделан - редкий товар
5) кнопки слайдера- доделаны
6) heder - доставка, гарантии, контакты - если на гл странице ок- также сделаю на стр каталога
7) стр каталога - форма фильтр, популярные
8) стр каталога хлебные крошки
9) кое-что не получается - напишу в telegram

TV37 committed:
Кнопка Каталог товаров активность

TV37 committed:
Продолжает добавлять активности к элементам.

+ исправляет код, где надо.

TV37 committed:
Окончание ДЗ Подготовка к защите

Director099 commented on styles/styles.css:
```diff
@@ -311,10 +509,9 @@ body {
-  margin-top: auto;
+  margin-top: 10px;
   gap: 20px 40px;
   padding: 14px 0 0 60px;
```
при ховере, когда пытаешься навести на подменю оно скрывается

лучше убрать `margin-top: 0`   и сделать `padding: 24px 0 0 60px;`
И будет все отлично

Director099 commented on index.html:
```diff
@@ -196,7 +157,7 @@ <h2 class=" visually-hidden popover-title">Корзина</h2>
           <ul class="navigation-service-menu">
             <li class="navigation-service-item">
-              <a class="navigation-service-link" href="#">Доставка</a>
+              <a class="navigation-service-link service-link-current" href="#">Доставка</a>
```
давай лучше уберем класс, в макете его нету, боюсь проверяющий наставник докапается

Director099 commented on styles/styles.css:
```diff
@@ -344,25 +541,61 @@ body {
 }
 
-.navigation-catalog-toggler:hover 
+.navigation-catalog-toggler:hover
```
При ховере плюс/минус  не меняется

Сделай лучше так 
```
.navigation-catalog-toggler:hover .navigation-toggler-minus-plus {
  background-image: none;
    position: absolute;
    content: " ";
    width: 100%;
    height: 2px;
    background-color: rgba(0 0 0 /100%);
    top: 7px;
}
```

Director099 commented on styles/styles.css:
```diff
@@ -888,51 +1246,49 @@ body {
-
-.category-item-img :active {
+.category-item:active {
   background-color: #FFE17F;
```
убери данный цвет т.к. такого состояния нет  http://joxi.ru/bmo8NXOtlpx4k2

Ну жно убрать только этот лишний цвет

Director099 commented on catalog_monopods.html:
```diff
@@ -375,9 +339,11 @@ <h2 class="visually-hidden">Карточки моноподов для селф
                     alt="любительская селфи-палка">
                 </a>
+                <button class="button product-card-button" type="button">Подробнее</button>
+                <span></span>
```
Не оставляй пожалуйста пучтые теги, это критерии

Director099 commented on catalog_monopods.html:
```diff
@@ -386,9 +352,12 @@ <h4 class="product-card-title">Любительская <br>селфи-палк
+                  <span class="new-product">NEW</span>
                 </a>
+                <button class="button product-card-button" type="button">Подробнее</button>
+                <span></span>
```
Удалить

Director099 commented on catalog_monopods.html:
```diff
@@ -397,9 +366,11 @@ <h4 class="product-card-title">Профессиональная <br>селфи-
                     alt="непотопляемая селфи-палка">
                 </a>
+                <button class="button product-card-button" type="button">Подробнее</button>
+                <span></span>
```
Удалить

Director099 commented on catalog_monopods.html:
```diff
@@ -408,9 +379,11 @@ <h4 class="product-card-title">Непотопляемая <br>селфи-пал
                     alt="селфи-палка «Следуй за мной»">
                 </a>
+                <button class="button product-card-button" type="button">Подробнее</button>
+                <span></span>
```
Удалить

Director099 commented on catalog_monopods.html:
```diff
@@ -397,9 +366,11 @@ <h4 class="product-card-title">Профессиональная <br>селфи-
+                  <img class="product-card-image" src="images/catalog-monopods/product-3.png" width="360" height="380"
                     alt="непотопляемая селфи-палка">
                 </a>
+                <button class="button product-card-button" type="button">Подробнее</button>
```
Что то ты сильно переделала, у тебя весь блок был ссылкой, это было самое правильное, а данную кнопку нужно было делать 

` <spanclass="button product-card-button" >Подробнее</span>` т.к. клик идет по ссылке 

Если долго переделывать то сделай лучше ссылкой

Director099 commented on styles/styles.css:
```diff
@@ -2320,34 +3090,98 @@ input[type="email"]:invalid {
 }
 
-/* пагинация каталога*/
+.product-card-link:hover + .product-card-button {
+  display: block;
+}
+
+.product-card > .button:hover:before {
+  height: 8px;
+}
+
+.product-card-link:active,
+.product-card-link:active ~ .product-card-text {
+  opacity: 0.3;
+}
+
+.product-card-link:active + .product-card-button {
+  display: block;
+}
+
+.product-card > .button:active:before {
+  height: 8px;
+}
+
+.product-card-link:focus ~ span:before {
+  content: " ";
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  outline: 2px solid  #AF4FFF;
+  outline-offset: 2px;
+}
```
У тебя при наведении на кнопку, она исчезает 
Сделай тогда hover/focus/ active на `.product-card` а не `.product-card-link`

TV37 committed:
Исправление ошибок

TV37 committed:
Исправляет ошибки по последнему ПР

TV37 committed:
Убирает не нужные атрибуты

1) атрибуты disabled, которые использовались для проверки состояния, но которых нет в макете

TV37 committed:
Доводит до идеала :focus

TV37 committed:
Справляет ошибки - проверка в Валидаторе
